itest/lnd_route_blinding_test.go

```diff
@@ -397,6 +401,145 @@ func setupFourHopNetwork(ht *lntest.HarnessTest,
 	}
 }
 
+// createBlindedRoute creates a blinded route to the recipient node provided.
+// The set of hops is expected to start at the introduction node and end at
+// the recipient.
+func (b *blindedForwardTest) createBlindedRoute(hops []*forwardingEdge,
+	dest *btcec.PublicKey, finalCLTV uint16) *routing.BlindedPayment {
+
+	// Create a path with space for each of our hops + the destination
+	// node. We include our passed final cltv delta here because blinded
+	// paths include the delta in the blinded portion (not the invoice).
+	blindedPayment := &routing.BlindedPayment{
```

Comment:
Perhaps we just just elevate this _entire_ routine into a `BuildBlindedRoute` RPC call? Similar to `BuildRoute.

Reply:
Happy to do this if it's blocking, but seems like it's a bit of a scope creep for a forwarding PR? 

Leaving them in the tests gives us future API flexibility when we're working on path creation/receiving rather than committing to something that (at least I) haven't fully thought through yet.

Reply:
Def not blocking, I think once we start to build out the higher level RPCs around sending+receiving w/ blinded paths we'd also flesh this out.